src/components/AframeScene.jsx

```diff
@@ -47,38 +66,38 @@ function AframeScene({
         cursor-listener
       />
 
-      <Entity
+      {/* MODEL */}
+      <a-entity
         id="volumeCube"
         class="clickableMesh"
-        loader={{
-          useTransferFunction: useTransferFunction,
-          colorMap: colorMap,
+        loader={toAframeString({
           alphaXDataArray: transferFunction.map((p) => p["x"]),
           alphaYDataArray: transferFunction.map((p) => p["y"]),
+          colorMap: colorMap,
```

Comment:
since this is an object, I think you could use this syntax (same with `useTransferFunction`)
```suggestion
          colorMap,
```

Reply:
That worked (: